### PR TITLE
Complete french translation in common

### PR DESCRIPTION
**Pull request by Pangoraw**

### CODE REVIEW

File: public/locales/fr/common.json [hunk 1]
```diff
@@ -1,7 +1,45 @@
...
+      "cut": "Couper",
+      "copy": "Copier",
+      "paste": "Coller"
+  },
+
+  "states": {
+    "disabled": "Désactivé",
+    "enabled": "Activé",
+    "loading": "Chargement...",
+    "connecting": "Connection...",
```
Comment (abclive) [inline public/locales/fr/common.json]: Terme faux. Une erreur commune en français on dit Connexion et non pas Connection (le terme anglais)

Comment (Pangoraw) [inline public/locales/fr/common.json]: Merci, je corrige ça.

**Comment by bilou84** [comment]
Thanks!

Please prefer to use English for discussion (event though it's about a French translation).
Also, try to have more explicit commit message. Something like "Fix typo in French translation" for instance ;)

**Comment by Pangoraw** [comment]
Thanks you ! I will know it for next time !

**Pull request merged**